{
    "checksum": "18ddf761ce5cd25e586413e4ebccde9b9cef956fda408acdbe0ab5aa8a39623b",
    "file": "conf.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/plain"
    },
    "number_of_duplicates": 2,
    "package": "python-hdf4",
    "path": "python-hdf4/0.10.3-1/doc/conf.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-hdf4/0.10.3-1/",
        "metric": {
            "size": 1000
        },
        "pts_link": "https://tracker.debian.org/pkg/python-hdf4",
        "sloc": [
            [
                "ansic",
                9543
            ],
            [
                "python",
                3744
            ],
            [
                "makefile",
                192
            ],
            [
                "sh",
                7
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian-gis-team/python-hdf4",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-hdf4/0.10.3-1/doc/conf.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 8182,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.10.3-1"
}